contentpacks/update_dubbed_video_spreadsheet.py

```diff
@@ -0,0 +1,375 @@
+            if title == khan_data.get("video_title"):
+                node["tutorial"] = khan_data.get("tutorial_title")
+                node["domain"] = khan_data.get("domain")
+                node["topic"] = khan_data.get("topic_title")
+                node["subject"] = khan_data.get("subject_title")
+    dump_json = os.path.join(BUILD_PATH, "video_node_data.json")
+    logging.info("Save dubbed_video data in %s" % dump_json)
+    with open(dump_json, "w") as f:
+        ujson.dump(node_data, f)
+                
+                
+
+def map_cell_range(start_col, end_col, start_row, end_row):
+    """Get the cell range to update the cells by batch"""
+    # Sample (A4:C4)
+    cell_range = '{col_i}{row_i}:{col_f}{row_f}'.format(
+        col_i=convert_number_to_column(start_col),
+        col_f=convert_number_to_column(end_col),
+        row_i=start_row,
+        row_f=end_row)
+    return cell_range
+
+
+def convert_number_to_column(n, b=string.ascii_uppercase):
+    """Convert the numbers into letters"""
+    d, m = divmod(n, len(b))
+    return convert_number_to_column(d - 1, b) + b[m] if d else b[m]
+
+
+def update_cell_by_batch(sheet, node_data,  lang_column, node_key, start_col, end_col, start_row, end_row):
+    """Update the cell by batch """
+    header_cell_range = map_cell_range(start_col=start_col, end_col=end_col, start_row=start_row, end_row=end_row)
+    logging.info("Updating cell range at %s" % header_cell_range)
+    title_cell_list = sheet.range(header_cell_range)
+    video_count = 0
+    for nodes, cell in zip(node_data, title_cell_list):
+        if node_key == "youtube_ids":
+            node_obj = nodes.get(node_key)
+            for key, video_obj in node_obj.items():
+                if key == lang_column.lower():
+                    video_count += 1
+                    cell.value = video_obj
+        else:
+            node_obj = nodes.get(node_key)
+            cell.value = node_obj
+            if cell.value is None:
+                cell.value = ""
+    if node_key == "youtube_ids":
+        logging.info("Total video count for %s: (%s)" % (lang_column, video_count))
+    sheet.update_cells(title_cell_list)
+  
+        
+def update_or_create_spreadsheet(spreadsheet=None):
+    """Map the node_data.json into the spreadsheet"""
+    dump_json = os.path.join(BUILD_PATH, "video_node_data.json")
+    logging.info("Lets update the spreadsheet using the %s" % dump_json)
+    with open(dump_json, 'r') as f:
+        node_data = ujson.load(f)
+        
+    spreadsheet_headers = list(SPREADSHEET_DEFAULT_VALUE)
+    node_obj_count = len(node_data)
+    for sup_lang in LE_SUPPORTED_LANG:
+        spreadsheet_headers.append(sup_lang.upper())
+    column_length = len(spreadsheet_headers)
+    try:
+        logging.info("Create spreadsheet %s with columns:(%s) and rows:(%s)" % (BUILD_VERSION, column_length, node_obj_count))
+        spreadsheet.add_worksheet(title=BUILD_VERSION, rows=node_obj_count, cols=column_length+2)
```

Comment:
I can see that `BUILD_VERSION` is in use. But you might wanna use the version specified in the command line argument instead?